website/docs/developer/research/research-lambda-container.md

```diff
@@ -148,6 +148,156 @@ Based on this research, it looks like using lambda as containers is possible whe
+  permissions_boundary = var.permissions_boundary_arn
+  tags                 = var.tags
+}
+
+
+resource "aws_iam_role_policy_attachment" "ecs_role_policy" {
+  role       = aws_iam_role.orca_ecs_task_execution_role.name
+  policy_arn = "arn:aws:iam::aws:policy/AdministratorAccess"
+}
+
+resource "aws_ecs_cluster" "test-cluster" {
+  name = "${var.prefix}_orca_ecs_cluster"
+  capacity_providers = ["FARGATE"]
+  default_capacity_provider_strategy {
+    capacity_provider = "FARGATE"
+    weight            = 100
+  }
+}
+
+# Defines how the image will be run. container_definitions can be replaced by data element.
+resource "aws_ecs_task_definition" "task" {
+  family                   = "${var.prefix}_orca_sqs_task"
+  network_mode             = "awsvpc"
+  requires_compatibilities = ["FARGATE"]
+  cpu                      = "4096"
+  memory                   = "8192"
+  task_role_arn            = aws_iam_role.orca_ecs_tasks_role.arn
+  execution_role_arn       = aws_iam_role.orca_ecs_task_execution_role.arn
+  container_definitions    = <<DEFINITION
+[
+  {
+    "name": "sqs_task_container",
+    "image": "236859827343.dkr.ecr.us-west-2.amazonaws.com/adorn-test-repo:latest",
+    "cpu": 4096,
+    "memory": 256,
+    "networkMode": "awsvpc",
+    "environment": [
+      {
+        "name": "TARGET_QUEUE_URL",
+        "value": "https://sqs.us-west-2.amazonaws.com/236859827343/doctest-orca-internal-report-queue.fifo"
+      }
+    ],
+    "logConfiguration": {
+      "logDriver": "awslogs",
+      "options": {
+        "awslogs-create-group": "true",
+        "awslogs-region": "us-west-2",
+        "awslogs-group": "${var.prefix}_orca_sqs_task",
+        "awslogs-stream-prefix": "ecs"
+      }
+    }
+  }
+]
+DEFINITION
+}
+:::note
+The above example was developed for deploying a simple task that posts to an sqs queue. Names and values should be changed to match new use cases.
+Values such as `cpu` and `memory` should similarly be reevaluated.
+:::
+:::warning
+Applying admin permissions to orca_ecs_task_execution_role is likely overly permissive.
+:::
+- The Fargate task can now be run in the ECS cluster.
+This can be done through [boto3](https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/ecs.html#ECS.Client.run_task) or the GUI,
+though the former is presently untested.
+- [Pricing](https://aws.amazon.com/ecs/pricing/?trk=2f064982-4fad-4e1f-ab75-e1df26258a60&sc_channel=ps&sc_campaign=acquisition&sc_medium=GC-PMM|PS-GO|Brand|All|PA|Database|ECS|Product|US|EN|Text|xx|SEM|PMO22-13405&s_kwcid=AL!4422!3!547620651289!e!!g!!amazon%20ecs%20pricing&ef_id=EAIaIQobChMIsO-ehu2l9wIVCfrICh0gOQ5OEAAYASABEgIZmfD_BwE:G:s&s_kwcid=AL!4422!3!547620651289!e!!g!!amazon%20ecs%20pricing) only applies to what is used in the moment, and can auto-scale down.
+  - Minimum compute time is one minute, so this architecture should not be used for small, frequent tasks.
+
+
+
```

Comment:
I would put recommendations in regards to the change (should we move forward or not) and link to any Jira cards to make the necessary changes. That should satisfy the acceptance criteria.

Reply:
Added and created.